lib/getters/orderGetters.ts

```diff
@@ -1,5 +1,8 @@
 const getSubmittedDate = (order) => order?.submittedDate
-const getProductNames = (order) => order?.products
+const getProductNames = (order) => {
+  const productNames = order?.items.map((product) => product.product.name)
```

Comment:
`product.product.name` should be `item.product.name`.

Reply:
Resolved